src/Pkg.jl

```diff
@@ -596,6 +597,12 @@ end
 # Precompilation #
 ##################
 
+function _auto_precompile(ctx::Types.Context)
+    if parse(Int, get(ENV, "JULIA_PKG_PRECOMPILE_AUTO", "1")) == 1
```

Comment:
Unrelated to the PR, but this should not error for bad input
```suggestion
    if tryparse(Int, get(ENV, "JULIA_PKG_PRECOMPILE_AUTO", "1")) == 1
```

Reply:
True. Updated